app/Services/FollowerService.php

```diff
@@ -212,4 +213,15 @@ public static function delCache($id)
 		Cache::forget(self::FOLLOWERS_SYNC_KEY . $id);
 		Cache::forget(self::FOLLOWING_SYNC_KEY . $id);
 	}
+
+	public static function localFollowerIds($pid, $limit = 0)
+	{
+		$key = self::FOLLOWERS_LOCAL_KEY . $pid;
+		$res = Cache::remember($key, 86400, function() use($pid) {
```

Comment:
Caching for a day seems a bit long? Shouldn't this maybe be an hour or few?

Reply:
Good point, I will update this!